x-pack/plugin/esql/qa/testFixtures/src/main/resources/rerank.csv-spec

```diff
@@ -82,6 +121,26 @@ book_no:keyword | title:text                                            | author
 ;
 
 
+reranker using multiple fields with some non text fields
+required_capability: rerank
+required_capability: match_operator_colon
+
+FROM books METADATA _score
+| WHERE title:"war and peace" AND author:"Tolstoy"
+| RERANK "war and peace" ON title, ratings WITH { "inference_id" : "test_reranker" }
+| EVAL _score=ROUND(_score, 2)
+| SORT _score DESC, book_no ASC
+| KEEP book_no, title, ratings, _score
+;
+
+book_no:keyword | title:text                                            | ratings:double | _score:double
```

Comment:
ℹ️ Combining text and non-text fields. Will be encoded in a YAML document that will be passed to the reranker.